### DISCUSSION TITLE
Help! Need some examples for Advanced Slides fork

### DISCUSSION THREAD

**Discussion by ebullient** [ebullient/obsidian-slides-extended#2] [General]
This repository is a fork of the original [Obsidian Advanced Slides plugin](https://github.com/MSzturc/obsidian-advanced-slides) by @MSzturc. 

It is a significant revision, with updates across the stack (including to the latest reveal.js).

I need some help, specifically, example source slides with the rendered html result (I don't need the supporting material, I just want to know what the html looked like). Especially for slides that were difficult to get working (splits, grids, etc.).

Add a comment to this thread with:

- an attached md/html file
- a list of plugins your slides use
- if you have custom css, I would love an example of that, too.

Please and thank you. 🙏

**Comment by batonac** [comment]
My slides aren't complicated, but I'm thrilled to see that development continues, and that you continue to support slide embedding. This is the feature that really sets this plugin apart IMO. Neither the built-in or the Marp slides plugin supports creating custom sequences by embedding slide collections into the current note. It's a wonderful feature and I hope it continues to enjoy first-class support.

**Reply by ebullient** [reply]
Examples that I can use (source + export) would really help me ensure things continue to work. :)

They don't have to be complicated.. they just have to exist. 

If you can capture the function you like best in a standalone example, that will help ensure it continues to function.

**Reply by batonac** [reply]
Here's the simplest scenario that I know how to capture. Several slide collections are in "Stacks" which are embedded in a custom presentation in "Decks". The default embed slash command doesn't include the full path as long as the note name is unique, but the full path is necessary for this to work with this plugin (at least at the moment), so that takes a bit more work to configure properly when folders are used.

Both the source and export are in the zip. If you'd like this in a different format, please let me know.
[Custom Slideshow.zip](https://github.com/ebullient/obsidian-slides-extended/files/14759205/Custom.Slideshow.zip)

**Comment by UUoocl** [comment]
Thanks for your work to improve Advanced Slides.  I created a plugin that adds OBS control to Advanced Slides.  
https://github.com/UUoocl/Obsidian_AdvancedSlides_Demo_Vault
This video explains how Obsidian Advanced Slides can control OBS via websocket messages. 
https://www.youtube.com/watch?v=P3NDWLdrQF0
Please consider including "OBS Control" as a feature in Obsidian Slides Extended. I think this feature is important for presenting Obsidian slides in virtual meetings and recording slide presentations.

**Comment by ebullient** [comment]
Thanks all. Will get there ;)
